.gitignore

```diff
@@ -1,15 +1,52 @@
 *.pyc
-!.travis.yml
 *.swp
 *.lock
 *.log
 *.pid
+cms/django
 *.DS_Store
+*.svn
+.*
+!.travis.yml
+!.coveragerc
+!.codeclimate.yml
+*.xml
+/*env*/
+*.sqlite
+cms/media/cms_page_media/
+cms/docs/build
+example/run
+example/local_settings.py
+/reversion/
+htmlcov
+build
+dist
+.ropeproject
+.project
+.pydevproject
+.settings
 /*.egg-info/
 /*.egg/
-.idea
-.tox
-env
-.coverage
-htmlcov/
-*~
\ No newline at end of file
+*.egg
+develop-eggs
+downloads
+eggs
+parts
+bin
+/dist
+*~
+/docs/_build
+/docs/*.html
+distribute-*.tar.gz
+include/
+lib/
+man/
+share/
+.editorconfig
```

Comment:
this particular line is something we should address in a separate PR, pretty much like here https://github.com/divio/django-cms/pull/4205 since editorconfig is very useful by itself